tests/Unit/Visualization/Python/CMakeLists.txt

```diff
@@ -44,6 +50,11 @@ if(${BUILD_PYTHON_BINDINGS})
     PROPERTIES
     TIMEOUT 10
     )
+  set_tests_properties(
+    "Unit.Visualization.Python.PlotMemoryMonitors"
+    PROPERTIES
+    TIMEOUT 40
```

Comment:
For some reason, the command itself takes a very long time to run. I wasn't able to determine why, even with `spectre --profile`. Any suggestions of where I'm going wrong are welcome.